src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/exceptions.php

```diff
@@ -1,12 +1,17 @@
 <?php
 
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 $container->loadFromExtension('framework', [
     'exceptions' => [
         BadRequestHttpException::class => [
             'log_level' => 'info',
             'status_code' => 422,
         ],
+        NotFoundHttpException::class => [
+            'log_level' => null,
+            'status_code' => 400,
+        ],
```

Comment:
Can we also add a test case when `status_code` set to `null`?

Reply:
Sure, I will add it later

Reply:
Done.

Additionally I changed allowed type for XML attribute in XSD schema too to allow specifying `null` https://github.com/symfony/symfony/pull/46956/files#r923095768